docs/configuration.rst

```diff
@@ -0,0 +1,26 @@
+Configuration Reference
+=======================
+
+``towncrier`` has many knobs and switches you can use, to customise it to your project's needs.
+The setup in the `Quick Start <quickstart.html>`_ doesn't touch on many, but this document will detail each of these options for you!
+
+For how to perform common customisation tasks, see `Customisation <customisation/index.html>`_.
+
+``[tool.towncrier]``
+--------------------
+
+All configuration for ``towncrier`` sits inside ``pyproject.toml``, under the ``tool.towncrier`` namespace.
+Please see `the TOML GitHub repo <https://github.com/toml-lang/toml>`_ for how to write TOML.
+
+Top level keys
+~~~~~~~~~~~~~~
+
+- ``package`` -- The package name of your project. (Python projects only)
```

Comment:
Is this required to be set for a Python project? What happens is not set for a python project?